.github/workflows/stale_pr.yml

```diff
@@ -0,0 +1,42 @@
+name: "Close stale pull requests"
```

Comment:
Just a note from our conversation: Let's hold off on adding this workflow until after we've triaged through all of the PRs.